src/main.jsx

```diff
@@ -1,10 +1,13 @@
-import React from 'react'
-import ReactDOM from 'react-dom/client'
-import App from './App'
-import './index.css'
+import React from "react";
+import ReactDOM from "react-dom/client";
+import App from "./App";
+import "./index.css";
+import { TodoProvider } from "./context/TodoContext";
 
-ReactDOM.createRoot(document.getElementById('root')).render(
+ReactDOM.createRoot(document.getElementById("root")).render(
   <React.StrictMode>
-    <App />
-  </React.StrictMode>,
-)
+    <TodoProvider>
+      <App />
+    </TodoProvider>
+  </React.StrictMode>
```

Comment:
In case you remove the React import as suggested above, import StricMode from React, like this:
```suggestion
  import { StrictMode } from "React"
  <StrictMode>
    <TodoProvider>
      <App />
    </TodoProvider>
  </StrictMode>
```